napari/utils/transforms/transforms.py

```diff
@@ -109,8 +110,8 @@ def _clean_cache(self):
         [self.__dict__.pop(p, None) for p in cached_properties]
 
 
-class TransformChain(EventedList, Transform):
-    def __init__(self, transforms=None) -> None:
+class TransformChain(EventedList[Transform], Transform):
+    def __init__(self, transforms: Optional[List[Transform]] = None) -> None:
```

Comment:
Base on superclass type annotation 
```suggestion
    def __init__(self, transforms: Optional[Iterable[Transform]] = None) -> None:
```

Reply:
👍  done